src/FirebaseController.js

```diff
@@ -363,6 +363,57 @@ class FirebaseController {
     }
   }
 
+  async getLovedRoutes(uid, okCallback, koCallback) {
+    try {
+      const q = query(collection(this.db, "routeLove"), where("uid", "==", uid))
+      const querySnapshot = await getDocs(q)
+      querySnapshot.docs.forEach(async (loveData)=>{
+        const data = loveData.data()
+
+        await this.getRoute(data.rid, false, (route) => {
+          route.rid = data.rid
+          okCallback(route)
+        }, koCallback)
+      })
+    } catch(e) {
+      koCallback && koCallback()
+    }
+  }
+
+  async getRouteLove(uid, rid, okCallback, koCallback) {
+    try {
+      const q = query(collection(this.db, "routeLove"), 
+        where("uid", "==", uid),
+        where("rid", "==", rid))
+      const querySnapshot = await getDocs(q)
+      okCallback && okCallback(querySnapshot.docs.length > 0, querySnapshot.docs[0].id)
+
+    } catch(e) {
+      koCallback && koCallback()
```

Comment:
if it can be resolved, we need to inform to the user as a 'Loved route not found/removed' and he has to be able to remove it